DatadogSessionReplay/Sources/PrivacyLevel+SessionReplay.swift

```diff
@@ -0,0 +1,79 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+import UIKit
+import DatadogInternal
+
+// MARK: - DatadogExtension for UIView
+
+/// Extension on `DatadogExtension` to provide access to `SessionReplayOverride` for any `UIView`.
+extension DatadogExtension where ExtendedType: UIView {
+    /// Provides access to Session Replay override settings for the view.
+    /// Usage: `myView.dd.sessionReplayOverride.textAndInputPrivacy = .maskNone`.
+    public var sessionReplayOverride: SessionReplayOverrideExtension {
+        return SessionReplayOverrideExtension(self.type)
+    }
+}
+
+// MARK: - Associated Keys
+
+private var associatedTextAndInputPrivacyKey: UInt8 = 3
+private var associatedImagePrivacyKey: UInt8 = 4
+private var associatedTouchPrivacyKey: UInt8 = 5
+private var associatedHiddenPrivacyKey: UInt8 = 6
+
+// MARK: - SessionReplayOverrideExtension
+
+/// `UIView` extension  to manage the Session Replay privacy override settings.
+public final class SessionReplayOverrideExtension {
+    private let view: UIView
+
+    public init(_ view: UIView) {
+        self.view = view
+    }
+
+    /// Text and input privacy override (e.g., mask or unmask specific text fields, labels, etc.).
+    public var textAndInputPrivacy: TextAndInputPrivacyLevel? {
+        get {
+            return objc_getAssociatedObject(view as AnyObject, &associatedTextAndInputPrivacyKey) as? TextAndInputPrivacyLevel
```

Comment:
/nit it's now redundant
```suggestion
            return objc_getAssociatedObject(view, &associatedTextAndInputPrivacyKey) as? TextAndInputPrivacyLevel
```

Reply:
Yes, I noticed and fixed it in the next PR :p